README.md

```diff
@@ -22,7 +22,7 @@ Note that the source generation for the services/client is all done by the `Akka
 
 ## Running
 
-Run this using [sbt](http://www.scala-sbt.org/). Since we want to run the application using HTTP/2 (which implies HTTPS)
+Since we want to run the application using HTTP/2 (which implies HTTPS)
```

Comment:
This sentence looks like it could be simplified further ;). Perhaps mention you do need to have sbt installed?

Reply:
Agreed. How about now?